package.json

```diff
@@ -4,6 +4,7 @@
   "version": "3.3.1",
   "homepage": "https://marionettejs.com/",
   "main": "lib/backbone.marionette.js",
+  "module": "lib/backbone.marionette.es.js",
```

Comment:
yes, that's what I meant. `lib/backbone.marionette.esm.js`
that would match https://github.com/rollup/rollup/wiki/pkg.module recommendation
sorry for the vagueness.